client/templates/globals/connectioninfo.js

```diff
@@ -1,5 +1,7 @@
 import { Meteor } from 'meteor/meteor'
 
+let isCollapsedWarningMessage = new ReactiveVar(false);
```

Comment:
It looks like you are toggling the visibility of the info-text via the style.display-attribute directly. So I think this reactive variable is not necessary. Unfortunately, the usage of the reactive variable is not correct (see for example line 18, line 38 or 46): If you change this variable via the equal-operator you set the type from ReactiveVar to boolean so you loose your reactive variable. Instead you have to use the get- and set-methods of the reactive variable class.

For further informations take a look at the docs: https://docs.meteor.com/api/reactive-var.html